homeassistant/components/venstar/strings.json

```diff
@@ -26,13 +26,22 @@
   "entity": {
     "sensor": {
       "schedule_part": {
+        "name": "Schedule Part",
```

Comment:
I don't think this works, I think you also need to:
- Update `VenstarSensorEntityDescription` to allow `name_fn` to be either `None` or a function returning a str
- Update `VenstarSensor.__init__` like this:
```diff
-        self._attr_name = entity_description.name_fn(sensor_name)
+        if entity_description.name_fn:
+            self._attr_name = entity_description.name_fn(sensor_name)
```